src/dynarec/la64/dynarec_la64_660f.c

```diff
@@ -1214,14 +1345,20 @@ uintptr_t dynarec64_660F(dynarec_la64_t* dyn, uintptr_t addr, uintptr_t ip, int
             nextop = F8;
             GETGX(v0, 1);
             GETEX(v1, 0, 0);
-            if (BOX64ENV(dynarec_fastnan)) {
-                VFMIN_D(v0, v0, v1);
-            } else {
+            // LoongArch FMIN/FMAX follow IEEE 754-2008 , it wll copy the none NaN value.
+            // If both NaN, then copy NaN.
+            // but x86 will copy the second if either v0[x] or v1[x] is NaN
+            if (!BOX64ENV(dynarec_fastnan) && v0 != v1) {
                 q0 = fpu_get_scratch(dyn);
-                VFCMP_D(q0, v0, v1, sLT);
```

Comment:
So we need to copy the second operand if unordered, can we just change `VFCMP_D(q0, v0, v1, sLT);` to `VFCMP_D(q0, v1, v0, cULT);` and reverse the selection?

Reply:
use cULE fix minpd/minps, use CUEQ + CLT fix maxpd/maxps